webapp/cas/go.mod

```diff
@@ -0,0 +1,12 @@
+module gopkg.in/c4dt/omniledger.v1/webapp/cas
```

Comment:
don't mix gopkg.in and go.mod - they're two different solutions to the same problem. It's `github.com/dedis/comniledger/webapp/cas`

Reply:
*c4dt ;)